rdr_service/model/genomics.py

```diff
@@ -450,6 +450,7 @@ class GenomicAW2Raw(Base):
     chipwellbarcode = Column(String(255), nullable=True)
     call_rate = Column(String(255), nullable=True)
     genome_type = Column(String(80), nullable=True)
+    alignment_pct_hg38 = Column(BigInteger, nullable=True)
```

Comment:
@j-kanuch,  this is the first `int` field we have in the `raw` models. Is this okay or would you like me to cast it as a string?  Wasn't sure if that was an inherent decision to have all the fields as strings?